### PR TITLE
[WIP] business_requirement_deliverable_report

### PR DESCRIPTION
**Pull request by martintorresvm**
---

Depends on:
- [ ] business_requirement_deliverable #3
  <a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/OCA/business-requirement/pull/7%23discussion_r67854373%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/7%23discussion_r67978619%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/7%23issuecomment-241676076%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/7%23issuecomment-243068439%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/OCA/business-requirement/pull/7%23issuecomment-241676076%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40victormartinelicocorp%20I%20think%20we%20should%20close%20this%20repo%20as%20there%20is%20now%20the%20new%20%20QWeb%20one.%5Cr%5CnWhat%20do%20you%20think%3F%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-08-23T09%3A25%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40elicoidal%20I%20close%20this%20PR%20that%20is%20replaced%20by%20%2315%20%20%22%2C%20%22created_at%22%3A%20%222016-08-29T08%3A50%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20d09456c073c055a1373517f9b1d67745f253f8f0%20business_requirement_deliverable_report/__openerp__.py%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/OCA/business-requirement/pull/7%23discussion_r67854373%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Where%20does%20%60report_docx%60come%20from%3F%5Cr%5CnWe%20should%20add%20a%20repo%20dependency%20for%20it.%22%2C%20%22created_at%22%3A%20%222016-06-21T11%3A56%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1246629%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dreispt%22%7D%7D%2C%20%7B%22body%22%3A%20%22True%3A%20we%20need%20to%20create%20a%20PR%20for%20base-report.%5Cr%5CnNevertheless%20we%20will%20probably%20rewrite%20the%20reports%20in%20qweb%20due%20to%20the%20limitation%20of%20this%20engine%20%28no%20html%20support%20which%20is%20annoying%20as%20all%20main%20fields%20are%20html%20content%29%22%2C%20%22created_at%22%3A%20%222016-06-22T00%3A49%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement_deliverable_report/__openerp__.py%3AL1-31%22%7D%7D%7D'></a>
  <a href='https://www.codereviewhub.com/'></a>
  - [ ] <a href='#crh-comment-Pull d09456c073c055a1373517f9b1d67745f253f8f0 business_requirement_deliverable_report/__openerp__.py 9'></a> &nbsp;<b><a href='https://github.com/OCA/business-requirement/pull/7#discussion_r67854373'>File: business_requirement_deliverable_report/**openerp**.py:L1-31</a></b>
  - <a href='https://github.com/dreispt'></a> Where does `report_docx`come from?
    We should add a repo dependency for it.
  - <a href='https://github.com/elicoidal'></a> True: we need to create a PR for base-report.
    Nevertheless we will probably rewrite the reports in qweb due to the limitation of this engine (no html support which is annoying as all main fields are html content)
  - [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/OCA/business-requirement/pull/7#issuecomment-241676076'>General Comment</a></b>
  - <a href='https://github.com/elicoidal'></a> @victormartinelicocorp I think we should close this repo as there is now the new  QWeb one.
    What do you think?
  - <a href='https://github.com/victormartinelicocorp'></a> @elicoidal I close this PR that is replaced by #15

<a href='https://www.codereviewhub.com/OCA/business-requirement/pull/7?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/OCA/business-requirement/pull/7?approve=1'></a>&nbsp;<a href='https://github.com/OCA/business-requirement/pull/7'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: business_requirement_deliverable_report/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Deliverable Report Docx Module',
+    'version': '9.0.1.0.3',
+    'category': 'report',
+    'depends': [
+        'report_docx',
```
Comment (dreispt) [inline business_requirement_deliverable_report/__openerp__.py:9]: Where does `report_docx`come from?
We should add a repo dependency for it.

Comment (elicoidal) [inline business_requirement_deliverable_report/__openerp__.py:9]: True: we need to create a PR for base-report.
Nevertheless we will probably rewrite the reports in qweb due to the limitation of this engine (no html support which is annoying as all main fields are html content)

**Comment by elicoidal** [comment]
@victormartinelicocorp I think we should close this repo as there is now the new  QWeb one.
What do you think?

**Comment by martintorresvm** [comment]
@elicoidal I close this PR that is replaced by #15

**Pull request closed without merging**